src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int AGE = 18;
+    public static final int PASSWORD = 6;
+    private RegistrationServiceImpl regService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+
+    @Test
+    void register_Login_NotOk() {
+        User testUser = new User();
+        user = new User();
+        testUser.setAge(20);
+        testUser.setLogin("Mersedes");
+        testUser.setPassword("king_1975");
+        storageDao.add(testUser);
+        user.setLogin("Mersedes");
+        assertThrows(RuntimeException.class, () ->
+                regService.register(user).equals(storageDao.get(user.getLogin())));
```

Comment:
```suggestion
    void register_sameLogin_NotOk() {
        User testUser = new User();
        testUser.setAge(20);
        testUser.setLogin("Mersedes");
        testUser.setPassword("king_1975");
        storageDao.add(testUser);
        User user = new User();
        user.setLogin(testUser.getLogin());
        user.setPassword("somepassword");
       user.setAge(25);
        assertThrows(RuntimeException.class, () ->
                regService.register(user).equals(storageDao.get(user.getLogin())));
```

Користувач, якого ми рееструемо з таким самим логіном, повинен мати всі інші валідні поля, інакше не відомо, чи RuntimeException виникла через те, що користувач з атким логіном вже є в базі, чи тому, що в користувача інші поля не валідні